python_bindings/src/PyVar.cpp

```diff
@@ -29,11 +29,6 @@ void define_var(py::module &m) {
             .def_static("outermost", &Var::outermost)
             .def("__repr__", &var_repr)
             .def("__str__", &Var::name)
-
-            // Cannot overload a method as both static and instance with PyBind11 and Python2.x
-            // .def_static("is_implicit", (bool (*)(const std::string &)) &Var::is_implicit)
```

Comment:
Does the comment imply that dropping python 2 support means we can uncomment these?

Reply:
No, this comment is in error -- PyBind11 doesn't support this out of the box. (We could probably find a way to make it work but since no one has complained about it so far, I'm inclined to just remove it permanently.)